### ISSUE TITLE
🐛 Bug: "New model ready for testing" issue is triggered before upload to DockerHub is completed

### ISSUE THREAD

**Issue by miquelduranfrigola** [ersilia-os/ersilia#959]
### Describe the bug.

In model eos7d58, a "New model ready for testing" issue was opened before upload to DockerHub was completed... https://github.com/ersilia-os/eos7d58/issues/2 Here you can see the failed dockerhub workflow: https://github.com/ersilia-os/eos7d58/actions/runs/7828896799/job/21359729719

The issue should only be opened when all other workflows are completed.

### Describe the steps to reproduce the behavior

Check the workflows in https://github.com/ersilia-os/eos7d58

### Expected behavior.

We should modify the YAML files in the https://github.com/ersilia-os/eos-template repository to open the issue only after upload to DockerHub has been done.

### Operating environment

Ubuntu 20.04 LTS

**Comment by GemmaTuron** [comment]
@miquelduranfrigola and @DhanshreeA 
This is as simple as removing the test issue part from the model_test and putting it at the end of the dockerhub upload. I agree this makes more sense.
Should we also add Hellen's suggestion to decrease the Docker image clearing cache?

**Comment by GemmaTuron** [comment]
Hi @miquelduranfrigola and @DhanshreeA 

Should I make the edit to the workflow files?

**Comment by miquelduranfrigola** [comment]
I think we can go ahead, yes

**Comment by DhanshreeA** [comment]
This is open and I can do this today. I think we should retroactively patch existing model repositories as well (in case we make a change in the future and need to re-test/upload new images?) But this will be low priority.

**Comment by GemmaTuron** [comment]
We have an automated script to update all workflows, but maybe let's do that once we are sure all issues with Ersilia's API keys changes are working - we could then to a model update at scale @DhanshreeA ?

**Comment by miquelduranfrigola** [comment]
Please note that there is this script in @DhanshreeA and @GemmaTuron . Is this useful? https://github.com/ersilia-os/ersilia/blob/master/.github/scripts/update_model_workflows_from_template.py

**Comment by DhanshreeA** [comment]
Addressed here: https://github.com/ersilia-os/eos-template/issues/48
